### PR TITLE
Implemented request rendering and post view.

### PR DESCRIPTION
**Pull request by monocliffe**
[TPOZ-1652](https://wpengine.atlassian.net/browse/TPOZ-1652) | [TPOZ-1653](https://wpengine.atlassian.net/browse/TPOZ-1653) | [TPOZ-1655](https://wpengine.atlassian.net/browse/TPOZ-1655)

Designed a React component to display the post index data. Implemented an axios request to the rails API on localhost to get the data. Rendered that data inside the React component.

### CODE REVIEW

File: src/Posts.js [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+import React from 'react'
+import axios from 'axios'
+
+export default class Posts extends React.Component {
+    state = {
+        posts: []
+    }
+
+    componentDidMount() {
+        axios.get('http://0.0.0.0:3000/posts.json')
```
Comment (tmwpengine) [inline src/Posts.js:10]: Not recommended to hardcode urls. Best to store these in the environment, or at the very least in a settings file that can be imported

**Review by tmwpengine — APPROVED** [review]
One comment aside this looks good to me, although I don't know a lot about React.

**Review by p-lu — APPROVED** [review]
LGTM, I would also look in to writing functional components instead of class components, which is a newer way of writing components that tends to be a bit cleaner

**Comment by monocliffe** [comment]
More work needed on feature.

**Pull request closed without merging**